### PR TITLE
fix(engine): yellow x runs walktrigger multiple times per tick

### PR DESCRIPTION
**Pull request by tannerdino**
matches:
- late 2006: https://youtu.be/Hsfn5HHtbSk?t=125
- early osrs: https://youtu.be/dKfFkEyqD8E?t=9

### CODE REVIEW

File: src/lostcity/engine/World.ts [hunk 1]
```diff
@@ -691,11 +691,12 @@ class World {
                             player.pathToPathingTarget();
                             continue;
                         }
-        
-                        player.pathToMoveClick(player.userPath, !Environment.NODE_CLIENT_ROUTEFINDER);
-    
-                        if (!player.opcalled && player.hasWaypoints()) {
-                            player.processWalktrigger();
+                        if (!Environment.NODE_CLIENT_ROUTEFINDER) {
```
Comment (ultraviolet-jordan) [inline src/lostcity/engine/World.ts:694]: This is probably breaking code. Test your code changes switching the env option on and off.

Comment (tannerdino) [reply src/lostcity/engine/World.ts:694]: Tested interactions, movement, walktriggers, and following, with both client and server pathing. All seems to function properly, not sure if theres a better way to implement this to match the videos

Comment (ultraviolet-jordan) [reply src/lostcity/engine/World.ts:694]: So why do you want this walktrigger here to run when you are only using server pathfinder and not client pathfinder? Then in the MoveClick packet you are running the walktrigger there both ways?

Comment (tannerdino) [reply src/lostcity/engine/World.ts:694]: I assumed that when osrs transitioned to server pathfinding they moved the walktrigger from the packet handler to client input. But that mightve not been the case. Looks to me that osrs's server pathfinding is calculated in the packet handler and not in client input. This [video](https://youtu.be/TnBJZD9AcJE?t=9) from oct 2016 shows that spam clicking whilst frozen runs the walktrigger multiple times per tick.

I think sometime later in osrs they changed moved the server pathfinding to client input (along with walktriggers)? Need to set the waypoints in the packet handler in order for commands like p_walk and p_stopaction to work properly in the walktrigger. Which is why I use pathToMoveClick in the handler.

**Pull request closed without merging**